.eslintcache

```diff
@@ -0,0 +1 @@
+[{"C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\index.js":"1","C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\serviceWorker.js":"2","C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\App.js":"3","C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\Question.js":"4","C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\Result.js":"5"},{"size":464,"mtime":1607810235118,"results":"6","hashOfConfig":"7"},{"size":5086,"mtime":1607810235121,"results":"8","hashOfConfig":"7"},{"size":1652,"mtime":1607810235112,"results":"9","hashOfConfig":"7"},{"size":787,"mtime":1607810235116,"results":"10","hashOfConfig":"7"},{"size":1027,"mtime":1607810235117,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"lg09hg",{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\index.js",[],"C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\serviceWorker.js",[],"C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\App.js",[],"C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\Question.js",[],"C:\\Users\\User\\Documents\\WebDevProjects\\hebrew-trainer\\src\\Result.js",[]]
```

Comment:
can you add `.eslintcache` to `.gitignore` and remove this file?

Reply:
👍